cookbooks/hdfsdu/recipes/hdfsdu_deploy.rb

```diff
@@ -0,0 +1,106 @@
+# Cookbook Name : hdfsdu
+# Recipe Name   : hdfsdu_deploy.rb
+# Description   : Download, configure and start hdfsdu
+
+Chef::Resource.send(:include, Hdfsdu::Helper)
```

Comment:
I could not find method `:extends` in `Chef::Resource`. If I replace `:include` with it, I get the following error: `undefined method 'extends' for Chef::Resource:Class`